docs/self-managed/platform-deployment/troubleshooting.md

```diff
@@ -2,26 +2,25 @@
 id: troubleshooting
 title: "Troubleshooting"
 sidebar_label: "Troubleshooting"
+description: "Troubleshooting considerations in Platform deployment."
 ---
 
 ## Keycloak requires SSL for requests from external sources
 
-When deploying the Camunda Platform to a provider it is important to confirm that the IP ranges used
-for container to container communication align with the IP ranges that Keycloak consider to be
-"local". By default, Keycloak considers all IPs outside those listed in their
+When deploying the Camunda Platform to a provider, it is important to confirm the IP ranges used
+for container to container communication align with the IP ranges Keycloak considers "local". By default, Keycloak considers all IPs outside those listed in their
 [external requests documentation](https://www.keycloak.org/docs/latest/server_installation/#_setting_up_ssl)
 to be external and therefore require SSL.
 
 As the [Camunda Platform Helm Charts](https://github.com/camunda/camunda-platform-helm) currently do
-not provide support for the distribution of the Keycloak TLS key to the other containers,
-for advice, we would recommend viewing the solution available in the
+not provide support for the distribution of the Keycloak TLS key to the other containers, we recommend viewing the solution available in the
 [Identity documentation](/docs/self-managed/identity/troubleshooting/common-problems#solution-2-identity-making-requests-from-an-external-ip-address).
 
 ## Identity redirect URL
 
-If HTTP to HTTPS redirection is enabled in the Load-balancer or Ingress, make sure to use the HTTPS
-protocol in the values file under `global.identity.auth.[COMPONENT].redirectUrl'.
-Otherwise you will get a redirection error in Keycloak.
+If HTTP to HTTPS redirection is enabled in the load-balancer or ingress, make sure to use the HTTPS
```

Comment:
The `Ingress` word here is a proper name. (refers to [Kubernetes Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/)).